### ISSUE TITLE
html-self-closing causes element after custom component to be ignored

### ISSUE THREAD

**Issue by gambolputty** [vuejs/eslint-plugin-vue#797]
Hi!

**My environment**	
* **ESLint version:** 3.15.0
* **eslint-plugin-vue version:** 5.1.0
* **Node version:**	8.10.0
* **Parcel version:** 1.11.0

**my configuration:**
eslintrc file:
```json
{
    "extends": [
      "standard",
      "plugin:vue/recommended"
    ],
    "plugins": [
        "standard",
        "promise"
    ],
    "env": {
        "browser": true
    },
    "parserOptions": {
        "ecmaVersion": 7,
        "sourceType": "module"
    },
    "globals": {
        "log": true,
        "warn": true,
        "error": true
    },
    "rules": {
        "no-const-assign": "warn",
        "no-this-before-super": "warn",
        "no-undef": "warn",
        "no-unreachable": "warn",
        "no-unused-vars": "warn",
        "constructor-super": "warn",
        "valid-typeof": "warn"
    }
}
```

**What did you do?**	
I have a self closing custom component ("css-selector-input") and a `<button>` following in one div. I'm using the Atom editor for linting. Running `node_modules/.bin/eslint src/components/modal/test.vue` did not give any output.
```vue
    <div>
      <css-selector-input
        v-model="selector"
        :results-count="false"
        :max="1"
        @debounced="triggertoggle"
      />
      <button
        type="button"
        @click.prevent="selectOther"
      >
        Select other element
      </button>
    </div>
```

**What did you expect to happen?**
I expected to see my custom component AND the button.

**What actually happened?**
I only saw my custom component. The button is nowhere in the DOM tree to be found.

**Comment by gambolputty** [comment]
Sorry, I saw this is the wrong repo. I'm closing this issue. Sorry!
